src/api/routes/ws/web-socket-transmitter.ts

```diff
@@ -23,23 +24,45 @@ export class WebSocketTransmitter {
   readonly db: PgStore;
   readonly server: http.Server;
   private channels: WebSocketChannel[] = [];
+  private queue: PQueue;
 
   constructor(db: PgStore, server: http.Server) {
     this.db = db;
     this.server = server;
+    // This queue will send all messages through web socket channels, one at a time.
+    this.queue = new PQueue({
+      autoStart: true,
+      concurrency: 1,
+      timeout: 10_000, // 10 seconds
```

Comment:
How does this timeout function? Does it mean 10 seconds in total to send updates to all registered sockets? Or 10 seconds per sockets? etc